velocity/pom.xml

```diff
@@ -73,7 +77,7 @@
                 <configuration>
                     <finalName>${outputName}</finalName>
                     <shadedArtifactAttached>true</shadedArtifactAttached>
-                    <minimizeJar>true</minimizeJar>
+                    <minimizeJar>false</minimizeJar>
```

Comment:
This can be removed completely as default is false

Reply:
Fixed in https://github.com/GeyserMC/Floodgate/pull/18/commits/e8d4e18cc87c4e699543aa967a36b17ac1306efb